packages/deploy-tooling/src/base/validate.ts

```diff
@@ -150,8 +152,10 @@ async function validateInputTextFormat(
     processInputValidationResult(result, output);
     result = validateClient(input.client);
     processInputValidationResult(result, output);
-    result = validateUrl(input.url);
-    processInputValidationResult(result, output);
+    if (!input.destination) {
```

Comment:
Do we have any code to sanitize the inputs? Something that will clear any URLs being passed in in this case. This code correctly skips the URL validation when a destination is available. However, there could be downstream code that could behave unpredictably if both a destination and (a potentially invalid) URL is being passed in.

Reply:
This validate module is only called when deploy is run in test mode. It is more for providing some helpful validation/diagnostic messages in the console. 

For actual deploymenet, we have [similar logic](https://github.com/SAP/open-ux-tools/blob/290b6b598103d4614c3c215f9d982b04d3b463bd/packages/system-access/src/base/connect.ts#L136) to validate the deploy target and create the correct AbapProvider.